app/src/behaviors/behavior_sensor_rotate_key_press.c

```diff
@@ -36,26 +73,22 @@ static int on_sensor_binding_triggered(struct zmk_behavior_binding *binding,
 
     switch (value.val1) {
     case 1:
-        keycode = binding->param1;
+        keycode_position = 1;
```

Comment:
This would cause every encoder to overlap with the second and third keys in the keymap.

I had a change of my own which was going to be part of a larger PR, but since it would fix this problem for you, I've split it out and created #1659. This gives each sensor a virtual key position which it can use for queueing behaviors.

Reply:
#1659 looks great! Thank you!